### PR TITLE
Module-1 Week-1 Project Portfolio: complete mobile first

### PR DESCRIPTION
**Pull request by tan12082001**
Microverse Module-1 project Portfolio: complete mobile-first
Introduced changes:
* Added My Works section and About me Section to the portfolio.
* Used CSS Grid and Flex-box properties for the section.
* HTML and CSS-based Webpage.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,6 +40,200 @@ <h2 id="green">I'm a software developer</h2>
...
+                                <ul>
+                                    <li>Database Management</li>
+                                    <li>Version Control</li>
+                                    <li>CLI</li>
+                                    <li>Web Development</li>
+                                    <li>API Design</li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to list the technology categories ("Languages", "Frameworks" and "Skills") container using the `ul` tag and make sure each one is wrapped into the `li` tag.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @tan12082001   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 
- Good UI :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (tan12082001) [reply index.html:233]: Hello, @levy002, thanks for reviewing my code. I have a doubt, do I need to make <h4> tags("Languages", "Frameworks", "Skills") as list elements or make the whole card(each card) as a list element? kindly suggest what I can do. 
Thank you.

Comment (tan12082001) [reply index.html]: Hello, @levy002 I understood the change specified. you made it clear in the sent mail I just had to read it again. thanks again.

Comment (levy002) [reply index.html]: Thanks @tan12082001 :+1:

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @tan12082001 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] No linter errors :white_check_mark: 
- [x] Pull request has a descriptive summary and title :white_check_mark: 
- [x] Repository has a professional readme file :white_check_mark: 
- [x] Implemented the changes from the previous reviewer :white_check_mark: 

## Optional suggestions
- [ Optional ] I think it would be better if you fix the overlap between the `header` and the `main` content on your page when users are scrolling through the page. The overlap currently blocks users from seeing the content on the page. Please see the screenshot below and rectify this.

Screenshot from 2023-06-02 21-33-52

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**